app/views/properties/_show.html.haml

```diff
@@ -13,3 +13,9 @@
           - t.add_row({th: t('properties.preferred_name')}, {td: display_in_multiple_languages(c.concept_properties[:label][:values])}) if  c.concept_properties[:label][:values].present?
           - t.add_row({th: t('properties.definitions')}, {td: display_in_multiple_languages(c.concept_properties[:definition][:values])}) if c.concept_properties[:definition][:values].present?
           - t.add_row({th: t('properties.parent')}, {td: display_in_multiple_languages(c.concept_properties[:parents][:values])}) if c.concept_properties[:parents][:values].present?
+
+      .property-content-format
```

Comment:
this block is already in the concept details component that you called up, why do it again here ?

Reply:
in the concept details, properties and instances, i just took the last part of the component which is the tabcontainer (json,xml,ntriples,turtle), the Raw data (html) part differ from these three

Reply:
Your change needs to be only the `concept details component` file.

Properties, Collections, Schemes, use it and it should work without changing each of them.